### ISSUE TITLE
kvm path is configurable only at build time

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#281]
Originally reported of Google Code with ID 225.

```
I see no reason why the KVM path should not be configurable at run-time, e.g. to switch between KVM and QEMU or between different KVM versions.
```
Originally added on 2012-03-24 20:09:20 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-07-19 14:56:30 +0000 UTC.

Added Labels: Type-Enhancement Hypervisor-KVM

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 18:16:06 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-04 09:02:02 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
